src/react-components/libraryDetailsView.tsx

```diff
@@ -13,6 +14,8 @@ export const LibraryDetailsView = ({ currentLibraryId }: LibraryDetailsViewProps
   const books = useAppSelector(state => state.books);
   const libraryDetails = useAppSelector(state => state.libraryIndex);
   const dispatch = useAppDispatch();
+  const [isFormVisible, setIsFormVisible] = useState(false);
```

Comment:
Good solution for tracking when the form is and isn't visible using the redux store!